src/main/java/core/basesyntax/impl/WritingToNewFileImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.WritingToNewFile;
+import core.basesyntax.db.FruitShopDao;
+import core.basesyntax.service.CantWorkWithThisFileException;
```

Comment:
The custom exception `CantWorkWithThisFileException` should be placed in a separate package, potentially `exception`, to organize the codebase better and follow the checklist.